openupgrade_scripts/scripts/payment_stripe/16.0.2.0/pre-migration.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2023 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_xmlids_renames = [
+    (
+        "payment_stripe.action_payment_acquirer_onboarding",
```

Comment:
Please check in this module and the other if the declarations of both records have different fields and one leaves a dangling value.

Reply:
The `action_payment_provider_onboarding` action of v16 (https://github.com/odoo/odoo/blob/16.0/addons/payment_stripe/views/payment_provider_views.xml) has the same data as `action_payment_acquirer_onboarding` of v15 (https://github.com/odoo/odoo/blob/15.0/addons/payment_stripe/views/payment_views.xml), only `name` and `res_model` changes.

The `payment_paypal` module does not have an onboarding action.

Reply:
OK, I meant the same fields set. You know that if for example the old action defines a domain and the new one not, the old domain prevails when migrating.

Reply:
Yes, exactly, when what you indicate happens, it is precisely what must be taken into account and left correct.